.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/FFmpeg/FFmpeg.git
 [submodule "thirdparty/zxing-cpp"]
 	path = thirdparty/zxing-cpp
-	url = https://github.com/Apra-Labs/zxing-cpp.git
+	url = https://github.com/zxing-cpp/zxing-cpp.git
```

Comment:
I will change that to point particular tag v.2.2.1

Reply:
Please test and verify this change Vishwanath

NOTE: Do not resolve the conversation from next time onwards, only the reviewer should do it

Reply:
@kumaakh,
I checked the latest Aprapipes code, one with vcpkg; there are pulling the latest zxing ( not the Aprapipes fork ).

Should we update the ZXING repo in Apra-Labs account, Instead of pointing to specific tag in ZXING original repository?